build.proj

```diff
@@ -134,6 +134,10 @@
   <Target Name="Clean">
     <Message  Importance="high" Text="Cleaning Cmdlets..." ContinueOnError="false" />
 
+    <!-- Clean out the NuGet cache -->
```

Comment:
Since we don't have tests of this in the common repo, can we add coverage to one of the existing Environment tests?

Reply:
@markcowl updated PR to include `LiveOnly` test